tests/jobs/scala/src/main/scala/TaskRunnerApp.scala

```diff
@@ -0,0 +1,33 @@
+import org.apache.spark.sql.SparkSession
+
+/**
+  * TaskRunnerApp creates a number of noop Spark tasks specified by 'numTasks' each of which
+  * sleeps for 'taskDurationSec'. The main goal of this class is to mimic long-running Spark
+  * applications (i.e. Spark Streaming) for the testing and troubleshooting purposes.
+  *
+  * Usage: TaskRunnerApp [numTasks] [taskDurationSec]
+  */
+object TaskRunnerApp {
+  def main(args: Array[String]) {
+    if (args.length < 2) {
+      println("Usage: TaskRunnerApp [numTasks] [taskDurationSec]")
+      System.exit(1)
+    }
+
+    val spark = SparkSession
+      .builder
+      .appName("TaskRunnerApp")
+      .getOrCreate()
+
+    val numTasks = args(0).toInt
+    val sleepMillis = args(1).toInt * 1000
+
+    spark.sparkContext.parallelize(0 until numTasks, numTasks).foreachPartition{_ =>
+      println(s"Sleeping for $sleepMillis")
```

Comment:
can you append `s` or `seconds` after `$sleepMillis`?

Reply:
added